pulp_smash/tests/pulp3/file/api_v3/test_crud_content_unit.py

```diff
@@ -0,0 +1,88 @@
+# coding=utf-8
+"""Tests that perform actions over content unit."""
+import unittest
+
+from requests.exceptions import HTTPError
+
+from pulp_smash import api, config, selectors, utils
+from pulp_smash.constants import FILE_URL
+from pulp_smash.tests.pulp3.constants import ARTIFACTS_PATH, FILE_CONTENT_PATH
+from pulp_smash.tests.pulp3.file.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+from pulp_smash.tests.pulp3.utils import clean_artifacts, get_auth
+
+
+class ContentUnitTestCase(unittest.TestCase, utils.SmokeTest):
+    """CRUD content unit.
+
+    This test targets the following issues:
+
+    * `Pulp #2872 <https://pulp.plan.io/issues/2872>`_
+    * `Pulp Smash #870 <https://github.com/PulpQE/pulp-smash/issues/870>`_
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variable."""
+        cls.artifact = {}
+        cls.cfg = config.get_config()
+        clean_artifacts(cls.cfg)
+        cls.content_unit = {}
+        cls.client = api.Client(cls.cfg, api.json_handler)
+        cls.client.request_kwargs['auth'] = get_auth()
+        files = {'file': utils.http_get(FILE_URL)}
+        cls.artifact = cls.client.post(ARTIFACTS_PATH, files=files)
+
+    @classmethod
+    def tearDownClass(cls):
+        """Clean class-wide variable."""
+        cls.client.delete(cls.artifact['_href'])
+
+    def test_01_create_content_units(self):
```

Comment:
If this test creates one content unit, shouldn't it be called `test_01_create_content_unit`, without the trailing "s"?